error-handlers.js

```diff
@@ -0,0 +1,12 @@
+exports.handles404 = (error, req, res, next) => {
+ if(error.match(/^\/api\//ig)) {
```

Comment:
You will not need this error.match if you are using '/ *' in app.js. _The '/ *' will handle any invalid path and should send a 404 and error message_. All of this for 404/invalid path can be done within your '/*' path on app.js :)